README.md

```diff
@@ -69,6 +68,20 @@ It has the following dependancies:
 - yojson
 - redis
 
+### Docker user ? 
+
+You can use the docker to build and run geneweb on any computer. 
```

Comment:
You can use Docker to build and run geneweb on any computer.

Reply:
oopsy 2